tests/functions.rs

```diff
@@ -403,3 +403,37 @@ fn test_path_exists_filepath_doesnt_exist() {
     .stdout("false")
     .run();
 }
+
+#[test]
+fn test_absolute_path_resolves() {
+  let test_object = Test::new()
```

Comment:
Take a look at other tests that use the `tree` function to populate the tmpdir with files. You should be able to use that to do everything in one expression.

Reply:
I'm not quite sure how to do that - could you explain how to add the `Tree` object to the `Test` then reference it in the `stdout()` builder call without getting borrow checker errors?

Reply:
Actually, looking at this again, I think that this would be tricky to do the way I had in mind, I think it's fine as-is.